.github/workflows/reusable-performance.yml

```diff
@@ -331,20 +331,13 @@ jobs:
       - name: Set commit details
         # Only needed when publishing results.
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/trunk' && ! inputs.memcached && ! inputs.multisite }}
-        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
-        id: commit-timestamp
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            const commit_details = await github.rest.git.getCommit({ owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha });
-            return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
+        run: echo "COMMITTED_AT=$(git show -s $GITHUB_SHA --format='%cI')" >> $GITHUB_ENV
```

Comment:
@swissspidy Since you're more familiar with GH Actions best practices, I'm curious whether there's any general preference between this (writing to an env variable for subsequent steps) vs what this was using before (a step output that is used in the following step to set the env variable).

Most importantly, can you confirm that this approach works, i.e. will `COMMITTED_AT` be available as an env variable in subsequent steps if it is written here?

Reply:
The rule of thumb is:

- if you want to make something available to a later step in the same job, add to `GITHUB_ENV`
- if you want to make something available from one job to another, use step outputs

So yes, adding to `GITHUB_ENV` makes this value available later in the same job.

Reply:
Thanks for clarifying!

Reply:
> The rule of thumb is:
> 
>     * if you want to make something available to a later step in the same job, add to `GITHUB_ENV`
> 
>     * if you want to make something available from one job to another, use step outputs
> 
> 
> So yes, adding to `GITHUB_ENV` makes this value available later in the same job.

Would it make sense to summarize this thinking inline before the step? I've been trying to include more inline documentation throughout action workflows to help make things more clear to contributors as much as possible. For example, [Core-59531](https://core.trac.wordpress.org/changeset/59531).

Reply:
```suggestion
       # Write to an environment variable to have the output available in later steps of the job.
        run: echo "COMMITTED_AT=$(git show -s $GITHUB_SHA --format='%cI')" >> $GITHUB_ENV
```